SITnett/templates/kontakt.html

```diff
@@ -28,7 +32,7 @@ <h3>Styret</h3>
 						{% else %}
 							<td><b>{{verv.plural|capfirst}}:</b></td>
 						{% endif %}
-						<td>
+						<td><div style="display: inline-block; white-space: nowrap">
```

Comment:
To ting som burde vurderes å se mer på før vi merger: En `<div>` inni en `<td>` er visstnok [ikke alltid en god idé] (selv om denne linken er veldig gammel)(https://stackoverflow.com/questions/1110915/is-a-div-inside-a-td-a-bad-idea), og inline-style er heller ikke helt heldig. Om denne stylingen er nødvendig burde vi få den inn i CSS-en @livejac

Reply:
Ja, jeg gjorde dette for å få en fornuftig oppførsel på tabellen, men det bør sikkert skrives bedre av dere. Vi kan snakkes om det i kveld!